vaadin-date-picker-flow-parent/vaadin-date-picker-flow/src/test/java/com/vaadin/flow/component/datepicker/DatePickerVariantTest.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright 2000-2021 Vaadin Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package com.vaadin.flow.component.datepicker;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+public class DatePickerVariantTest {
+
+    private final DatePicker datePicker = new DatePicker();
```

Comment:
Please initialize the instance in a setup method, reusing the same instance between tests is not a good practice and can lead to issues that are hard to debug.

Reply:
It is cargo cult from https://github.com/vaadin/flow-components/blob/master/vaadin-dialog-flow-parent/vaadin-dialog-flow/src/test/java/com/vaadin/flow/component/dialog/DialogThemeVariantTest.java

Reply:
Similarly, existing tests not using best-practices is not a good reason to not apply best practices to new tests. Let's fix it.

Reply:
As contributor I can rely only on what is already there ... but fixed as requested now.